onnx/defs/math/defs.cc

```diff
@@ -441,18 +472,8 @@ ONNX_OPERATOR_SET_SCHEMA(
             "T",
             {"tensor(float16)", "tensor(float)", "tensor(double)"},
             "Constrain input and output types to floating-point tensors.")
-        .FunctionBody(FunctionBodyHelper::BuildNodes(
-            {// nodes: {outputs, op, inputs, attributes}
-             FunctionBodyHelper::NodeDef{{"alpha"},
-                                         "Constant",
-                                         {},
-                                         {MakeRefAttribute(
-                                             "value_float",
-                                             "alpha",
-                                             AttributeProto::FLOAT)}},
-             {{"X_alpha"}, "Div", {"X", "alpha"}},
-             {{"Elu_Result"}, "Elu", {"X_alpha"}, {{"alpha", 1.f}}},
-             {{"Y"}, "Mul", {"alpha", "Elu_Result"}}})));
+        .SetContextDependentFunctionBodyBuilder(BuildContextDependentFunctionBodyCelu)
```

Comment:
I thought the plan was to restrict the type to only float, is that right? Should line 442/473 be changed to allow T to be only tensor(float)?